tests/chainer_tests/links_tests/theano_tests/test_theano_function.py

```diff
@@ -13,6 +14,9 @@
 
 
 @testing.with_requires('theano')
+@pytest.mark.skipif(
```

Comment:
`testing.without_requires` could be used.

Reply:
Fixed.